lib/tasks/valid_test_data_generator.rb

```diff
@@ -9,13 +9,13 @@ class LeadProviderPopulater
     include ActiveSupport::Testing::TimeHelpers
 
     class << self
```

Comment:
@leandroalemao this script is quite old Im not too sure about using it, maybe we should use the new seeds for setting up the schedules/statements etc manually and then enrolling some random ects/mentors using the services from the schools team, let's have a chat I think before proceeding